ee/common-annotations-1.3.adoc

```diff
@@ -0,0 +1,49 @@
+= Common Annotations 1.3  
+:author:            Tomaz Cerar
+:email:             tcerar@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+Commons Annotations 1.3 is a small change update to 1.2 MR.
+Main change is allowing @Repeatble on @DatasourceDefintion and @Priority annotation is allowed to also on methods now.
```

Comment:
`@Priority` is only for `ElementType.TYPE` and `ElementType.PARAMETER`. Perhaps remove the 'allowed on methods part.'